lms/templates/instructor/instructor_dashboard_2/certificates.html

```diff
@@ -0,0 +1,58 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%page args="section_data"/>
+<div class="certificates-wrapper">
+
+    <div class="example-certificates">
+        <h2>${_('Example Certificates')}</h2>
+
+        <div class="generate-example-certificates-wrapper">
+            <p>${_('Generate example certificates for the course.')}</p>
+
+            <form id="generate-example-certificates-form" method="post" action="${section_data['urls']['generate_example_certificates']}">
+                <input type="hidden" name="csrfmiddlewaretoken" value="${csrf_token}">
+                <input type="submit" id="generate-example-certificates-submit" value="${_('Generate Example Certificates')}"/>
+            </form>
+        </div>
+
+        % if section_data['example_certificate_status'] is not None:
+        <div class="example-certificate-status-wrapper">
+            <p>${_("Status:")}</p>
+            <ul>
+            % for cert_status in section_data['example_certificate_status']:
+                % if cert_status['status'] == 'started':
+                    <li>${_('Certificate "{name}" is generating').format(name=cert_status['description'])}</li>
+                % elif cert_status['status'] == 'error':
+                    <li>${_('An error occured while generating certificate "{name}": {error}').format(name=cert_status['description'], error=cert_status['error_reason'])}</li>
```

Comment:
Error generating example {name} certificates

Same comments as above re: quotation marks and capitalization.